setup.md

```diff
@@ -115,4 +115,28 @@ output similar to the below:
 git version 2.40.0
 ```
 
+## Configure Git to use your preferred text editor
+
+Many Git commands require you to provide input via a text editor (though you can often skip this step with additional command-line arguments). For example, running `git commit` by itself will open a text editor to allow you to enter a commit message. Depending on your OS and how you installed Git, the text editor Git will use can be any number of things, but is likely to be either [nano] or [vim], which are both terminal-based editors. It is best to configure your preferred editor explicitly as vim in particular is famously challenging for beginners.
```

Comment:
```suggestion
Many Git commands require you to provide input via a text editor (though you can often skip this step with additional command-line arguments). For example, running `git commit` by itself will open a text editor to allow you to enter a commit message. Depending on your OS and how you installed Git, the text editor Git will use can be any number of things, but is likely to be either [nano] or [vim], which are both terminal-based editors. It is best to configure your preferred editor explicitly (if you are using editor for the first time, it is advisable to use nano).
```

Reply:
Slightly rephrased the last sentence - I recall this is something I realised during [Carpentries Instructor Training](https://carpentries.github.io/instructor-training/08-motivation.html#how-can-content-influence-motivation)

Reply:
Cool. Thanks!